dev/run_e2e_load_tests.sh

```
@@ -0,0 +1,509 @@
+    line="${line#"${line%%[![:space:]]*}"}"
+
+    pid="${line%% *}"
+    command="${line#"$pid"}"
+    command="${command#" "}"
+
+    if ! report_server_command_matches "$command"; then
+      continue
+    fi
+
+    port=""
+    if [[ "$command" =~ $REPORT_SERVER_MODULE[[:space:]]+([0-9]+) ]]; then
```

Shellcheck (0.11.0)

[error] Use braces when expanding arrays, e.g. ${array[idx]} (or ${var}[.. to quiet).

(SC1087)